app/Http/Controllers/TeamController.php

```diff
@@ -39,13 +42,21 @@ public function store(Request $request)
         $validated = $request->validate([
             'name' => 'required|max:20', 
         ]);
-        $owner_id = auth()->id();
-        $team = new Team($validated);
-        $team->owner_id = $owner_id;
-        $team->save();
-        return to_route('manager.teams.show', $team)->with('success', 'チームを作成しました');
+        $user = Auth::user();
+        $user_id = $user->id;
+        $team = DB::transaction(function() use ($validated,$user_id){
+            $team = new Team($validated);
+                    $team->owner_id = $user_id;
+                    $team->save();
+                    $member = new Member();
+                    $member->team_id = $team->id;
+                    $member->user_id = $user_id;
+                    $member->role = 1;
+                    $member->save();
+                    return $team;
```

Comment:
ここは字下げしなくて大丈夫ですね。字下げする時は、`;` が登場せずに一行であることを明瞭にしている意図なので、それぞれ一行ごとなら特に字下げせずでOKです

```suggestion
            $team = new Team($validated);
            $team->owner_id = $user_id;
            $team->save();
            $member = new Member();
            $member->team_id = $team->id;
            $member->user_id = $user_id;
            $member->role = 1;
            $member->save();
            return $team;
```

Reply:
ご指摘ありがとうございます！